UnityProject/Assets/Scripts/Systems/Antagonists/Antags/CodeWordManager.cs

```diff
@@ -0,0 +1,59 @@
+using System.IO;
+using UnityEngine;
+using Mirror;
+
+namespace Antagonists
+{
+	public class CodeWordManager : NetworkBehaviour
+	{
+		public readonly SyncList<string> Words = new SyncList<string>();
+		public readonly SyncList<string> Responses = new SyncList<string>();
```

Comment:
only issue I see is hacked clients being able to see what the Traitor codewords are ,But I don't think it's too bad

Reply:
They wont get the highlight or anything unless theyve hacked the game to say they are traitor, at which point the game will treat them as traitor for all purposes anyways.